src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BlockSizeRule.cs

```diff
@@ -25,7 +25,7 @@ public override Task RunAsync(RuleContext context)
             // large by filling up the coinbase witness, which doesn't change
             // the block hash, so we couldn't mark the block as permanently
             // failed).
-            if (this.GetBlockWeight(context.BlockValidationContext.Block, options, context.Consensus.NetworkOptions) > options.MaxBlockWeight)
+            if (this.GetBlockWeight(context.BlockValidationContext.Block, options, TransactionOptions.All) > options.MaxBlockWeight)
```

Comment:
How do you know this is `TransactionOptions.All`?

Reply:
The default on context.Consensus.NetworkOptions was .All and it was not changed anywhere AFAICT

Reply:
I rolled back to be exactly as before the NetworkOptions was introduced.